### PR TITLE
Projects refresh fix on app upload

### PR DESCRIPTION
**Pull request by future-pirate-king**

### CODE REVIEW

File: app/components/project-list/index.js [hunk 1]
```diff
@@ -1,41 +1,34 @@
...
-import { filterPlatformValues } from 'irene/helpers/filter-platform';
 import { INPUT } from 'irene/utils/constants';
-import { PaginationMixin } from '../../mixins/paginate';
+import { DEFAULT_PARAMS } from 'irene/services/project';
 
-export default class ProjectListComponent extends PaginationMixin(Component) {
+export default class ProjectListComponent extends Component {
   @service intl;
   @service organization;
   @service realtime;
```
Comment (Yibaebi) [inline app/components/project-list/index.js:15]: Realtime service is not getting used.

File: app/components/submission-list/details/index.ts [hunk 2]
```diff
@@ -23,6 +39,18 @@ export default class SubmissionListDetailsComponent extends Component<Submission
         return 'is-progress';
     }
   }
+
+  @action
+  refreshProjectList() {
+    if (
+      this.router.currentRouteName === 'authenticated.projects' &&
```
Comment (Yibaebi) [inline app/components/submission-list/details/index.ts:46]: We can combine the if statement on line 24 of this file with this check also.

File: app/services/project.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+import ProjectModel from 'irene/models/project';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+type ProjectQueryResponse = DS.AdapterPopulatedRecordArray<ProjectModel> & {
+  meta?: { count: number };
+};
+
+export const DEFAULT_PARAMS = {
```
Comment (Yibaebi) [inline app/services/project.ts:18]: Can we have a better name for this to correlate with the project list query?

Comment (future-pirate-king) [reply app/services/project.ts:18]: done

File: app/components/project-list/index.js (hunk 1 above)
Comment (future-pirate-king) [reply app/components/project-list/index.js:15]: removed

**Review by Yibaebi — APPROVED** [review]

**Pull request merged**